frontend/src/__tests__/cypress/cypress/tests/e2e/dataScienceProjects/models/testDeployOCIModel.cy.ts

```diff
@@ -0,0 +1,89 @@
+import { deleteOpenShiftProject } from '~/__tests__/cypress/cypress/utils/oc_commands/project';
+import { HTPASSWD_CLUSTER_ADMIN_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
+import { projectListPage } from '~/__tests__/cypress/cypress/pages/projects';
+import {
+  retryableBefore,
+  wasSetupPerformed,
+} from '~/__tests__/cypress/cypress/utils/retryableHooks';
+import { createCleanProject } from '~/__tests__/cypress/cypress/utils/projectChecker';
+import { addConnectionModal, connectionsPage } from '~/__tests__/cypress/cypress/pages/connections';
+import {
+  inferenceServiceModal,
+  modelServingGlobal,
+  modelServingSection,
+} from '~/__tests__/cypress/cypress/pages/modelServing';
+import { checkInferenceServiceState } from '~/__tests__/cypress/cypress/utils/oc_commands/modelServing';
+
+const projectName = 'test-oci-deployment';
+const connectionName = 'test-oci-connection';
+const secretDetailsFile = Cypress.env('OCI_SECRET_DETAILS_FILE');
+const OCIRegistryHost = 'quay.io';
+const modelDeploymentURI = Cypress.env('OCI_MODEL_URI');
+const modelDeploymentName = 'test-oci-model-deployment';
```

Comment:
Maybe some of this can go to Fixture resources ?

Reply:
Sure I don't mind doing that. I did it on a previous PR, but I'm just wondering why we are doing that approach? It seems each fixture only works for one file, so I don't see the difference between fixture or const

Reply:
That is true, but the best practice would be to distinguish between test data and test logic. 
This allows us to control and reuse test data later, for example, to know if the same resource name could or could not be used in another test.